index.html

```diff
@@ -297,13 +301,18 @@ <h2>
         <a>Navigator</a> object's <a>battery promise</a> and abort these steps.
         </li>
         <li>If this <a>Navigator</a> object's <a>relevant global object</a>'s
-        <a>associated <code>Document</code></a>'s <a>browsing context</a>'s <a>
-          active document</a>'s <a>origin</a> is not <a>same origin-domain</a>
-          with the <a>origin</a> of the <a>current settings object</a> of this
-          <a>Navigator</a> object, then reject this <a>Navigator</a> object's
-          <a>battery promise</a> with a "<a>SecurityError</a>"
-          <a>DOMException</a>, return this <a>Navigator</a> object's <a>battery
-          promise</a> and abort these steps.
+        <a>associated <code>Document</code></a> is not <a>allowed to use</a>
+        the <code>battery</code> feature, then reject this <a>Navigator</a>
+        object's <a>battery promise</a> with a "<a>SecurityError</a>"
+        <a>DOMException</a>, return this <a>Navigator</a> object's <a>battery
+        promise</a> and abort these steps.
+          <div class="note">
+            In other words, this step rejects if the <a>associated
+            <code>Document</code></a>'s <a>browsing context</a>'s <a>active
+            document</a>'s <a>origin</a> is not <a>same origin-domain</a> with
+            the <a>origin</a> of the <a>current settings object</a> of this
+            <a>Navigator</a> object.
```

Comment:
You could add something like "unless specifically allowed by the document's feature policy." to this, if you wanted to be clear that it's *possible* to grant access to cross-origin subframes, but you have to be deliberate about it. The default situation is exactly as you describe here.